src/components/GameProvider/helpers/handleGameOut.js

```diff
@@ -0,0 +1,37 @@
+import { gameOut } from "../../../utils/constants";
+
+export const handleGameOut = (
+	selectedGameOut,
+	currPlayerScore,
+	calculatedValue, 
+	currPlayerHits,
+	hitValueString,
+	lastHistoryItem
+	) => {
```

Comment:
Kod funkcija koje primaju veci broj parametara mislim da je daleko lakse definisati sve parametre kao vrednosti unutar  objekta jer na taj nacin ne mora da se vodi racuna o redosledu po kojem su parametri prosledjeni pri pozivu funkcije.
```suggestion
export const handleGameOut = ({
	selectedGameOut,
	currPlayerScore,
	calculatedValue, 
	currPlayerHits,
	hitValueString,
	lastHistoryItem
	}) => {
```

Reply:
Parametri su sada proslijeđeni kao objekat.